curator/utils.py

```diff
@@ -865,7 +865,8 @@ def get_client(**kwargs):
         if kwargs['aws_key']:
             # Override these kwargs
             kwargs['use_ssl'] = True
-            kwargs['verify_certs'] = True
```

Comment:
You can probably just leave this line as it is, and then add the two new lines to override the `kwargs['verify_certs'] = True` if `ssl_no_validate` is `True`.  Otherwise, you'll need an `else` statement to assign it to `True`.

Reply:
🤦‍♂️ Yes, completely right, sorry.

Actually, just out of curiosity, as they are booleans we probably can just use `kwargs['verify_certs'] = not kwargs['ssl_no_validate']`

Reply:
True.  But I like the code readability (for new users) of a default being set, and then overridden.

This will all get cleaned up for version 6 anyway, though.